.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @jGaboardi @knaaptime @slumnitz
\ No newline at end of file
+* @martinfleis @jGaboardi @knaaptime @slumnitz
```

Comment:
@ocefpaf Seeing this weird error again that @martinfleis does not exist. Seems to be a rare error, but [others have experienced it](https://github.com/corona-warn-app/cwa-documentation/issues/911).

Reply:
I am curious if that can be ignored and we can merge? Or should something change here?

Reply:
I think we can ignore.